aspnetcore/host-and-deploy/linux-nginx.md

```diff
@@ -179,12 +179,12 @@ Once the Nginx configuration is established, run `sudo nginx -t` to verify the s
 To directly run the app on the server:
 
 1. Navigate to the app's directory.
-1. Run the app's executable: `./<app_executable>`.
+1. Run the app: `dotnet <app_assembly.dll>`, where `app_assembly.dll` is the assembly file name of the app.
 
 If a permissions error occurs, change the permissions:
 
 ```console
-chmod u+x <app_executable>
+chmod u+x <app_assembly.dll>
```

Comment:
🤔 This doesn't seem right. You don't require executable bit set on your app dll.

Reply:
mmm... hum ... I vaguely remember doing this tho.

It says "If ..." as a lead-in. Isn't that ok just in case?

Reply:
Just verified that the executable is not required

Reply:
Let's remove it from the doc